framework/assets/yii.js

```diff
@@ -219,9 +216,10 @@ window.yii = (function ($) {
                 }
                 if (!/(get|post)/i.test(method)) {
                     $form.append($('<input/>', {name: '_method', value: method, type: 'hidden'}));
-                    method = 'POST';
+                    method = 'post';
```

Comment:
- Use lower case. 
- With putting part for the existing form in `if`` check we also need to explicitly set `method` attribute.